types/fuzzyset/index.d.ts

```diff
@@ -0,0 +1,14 @@
+// Type definitions for fuzzset 1.0
+// Project: https://github.com/axiak/fuzzyset
```

Comment:
That's a python library!
The `fuzzyset` on NPM claims to have a homepage at http://github.com/washt/fuzzyset .
But that page claims to be published as `fuzzyset.js`, which is also a real NPM package.
So, what exactly are these types referring to? :confounded: